plugins/factorymod-paper/src/main/java/com/github/igotyou/FactoryMod/utility/IOConfigSection.java

```diff
@@ -87,7 +87,7 @@ private Clickable getIoClickable(Material adjacentType, Direction dir, String di
                 .append(Component.text(dirState.displayName).color(TextColor.color(dirState.color)))
                 .asComponent());
         ItemUtils.addComponentLore(display,
-            Component.text("L/M/R click to toggle I/F/O")
```

Comment:
LS as Left Shift is a little ambiguous here, how can we make this more understandable?

Reply:
I could change it to L+LS. Or move all of them from L/R to Left/Right to be even more clear

Reply:
How about we make this three lines, one for each option so they don't need to be abbreviated?